documentation/asciidoc/computers/remote-access/network-boot-raspberry-pi.adoc

```diff
@@ -249,7 +249,7 @@ Now monitor the `dnsmasq` log:
 
 [,bash]
 ----
-tail -F /var/log/daemon.log
+journalctl -ef
```

Comment:
I don't think the `e` flag is needed here. `tail` would normally print out the last 10 lines. With `e`, `journalctl` prints out the last 1000. Same for the other use of `-ef` in this PR - just `journalctl -f` would be equivalent of `tail -f`

Reply:
@jehlers42 Can you test this and check, and make the change if needed. Thanks.

Reply:
Confirmed that -f is all that's needed.